draft/0002-authentication.md

```diff
@@ -153,8 +168,9 @@ Resource groups may be marked as authenticated using the following:
 
 ### Resources
 
-Similiary to resource groups, a resource and it's children actions and examples
-can be marked as authenticated by placing an `Authenticated` list item.
+Similiary to resource groups, a resource and it's children actions and
```

Comment:
Now that I think about it, The phrase `Similiary to resource groups` sounds a bit weird.